ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.view.js

```diff
@@ -131,15 +131,31 @@ module.exports = Marionette.LayoutView.extend({
       'value'
     )[0]
     let title = this.basicTitle.currentView.model.getValue()[0]
-    if (title === '') {
-      let $validationElement = this.basicTitle.currentView.$el.find(
-        '> .property-label .property-validation'
-      )
-      $validationElement
-        .removeClass('is-hidden')
-        .removeClass('is-warning')
-        .addClass('is-error')
-      $validationElement.attr('title', 'Name field cannot be blank')
+    const $titleValidationElement = this.basicTitle.currentView.$el.find('> .property-label .property-validation')
+    const $attributeValidationElement = this.basicAttributeSpecific.currentView.$el.find('> .property-label .property-validation')
+    const titleEmpty = title === ''
+    const attributesEmpty = descriptors.length < 1
+    if (titleEmpty || attributesEmpty) {
```

Comment:
✏️ This if statement can be removed

Reply:
@willwill96 this was the only way I could think of to ensure that the warning symbol would appear for both cases before the method returns. If I remove this if and put separate return statements for each case, trying to save a result form with both required fields blank would only result in the warning symbol for the first case. The user would then enter a value for the first field, attempt to save, and get another warning symbol for the other required field. I thought it would be more straightforward to display both at once.

Reply:
👍 Counted my brackets wrong